material-overrides/main.html

```diff
@@ -30,7 +30,7 @@
 
 {% block announce %}
   <p>
-    We will be <a href="https://kluster.ai/blog/end-of-life-for-inference-fine-tuning-and-dedicated-deployments" target="_blank" rel="noopener">sunsetting Inference, Fine-Tuning, and Dedicated Deployments</a> on July 24 at 11pm ET.
+    Inference, Fine-Tuning, and Dedicated Deployments services have been sunset as we focus on Guardrails for AI. <a href="https://kluster.ai/blog/end-of-life-for-inference-fine-tuning-and-dedicated-deployments" target="_blank" rel="noopener">Find out more</a>.
```

Comment:
Mmm, I think we should still have the date in there. Also you didn't want to use "Learn more", but "Find out more" is quite similar? 🙂